deltametrics/plan.py

```diff
@@ -717,6 +717,85 @@ def compute_shoreline_length(shore_mask, origin=[0, 0], return_line=False):
         return length
 
 
+def compute_shoreline_distance(shore_mask, origin=[0, 0],
+                               return_distances=False):
+    """Compute mean and stddev distance from the delta apex to the shoreline.
+
+    Algorithm computes the mean distance from the delta apex/origin to all
+    shoreline points.
+
+    .. important::
+
+        This calculation is subtly different than the "mean delta radius",
+        because the measurements are not sampled evenly along the opening
+        angle of the delta.
+
+    .. note:: uses `np.nanmean` and `np.nanstd`.
+
+    Parameters
+    ----------
+    shore_mask : :obj:`~deltametrics.mask.ShorelineMask`, :obj:`ndarray`
+        Shoreline mask. Can be a :obj:`~deltametrics.mask.ShorelineMask`
+        object, or a binarized array.
+
+    origin : :obj:`list`, :obj:`np.ndarray`, optional
+        Determines the location from where the distance to all shoreline
+        points is computed.
+
+    return_distances : :obj:`bool`
+        Whether to return the sorted line as a second argument. If True, a
+        ``Nx2`` array of x-y points is returned. Default is `False`.
+
+    Returns
+    -------
+    length : :obj:`float`
+        Shoreline length, computed as described above.
+
+    line : :obj:`np.ndarray`
+        If :obj:`return_line` is `True`, the shoreline, as an ``Nx2`` array of
+        x-y points, is returned.
```

Comment:
Thanks for catching the docstring :roll_eyes: oops

Yeah, I think you're right. At first I was thinking that if someone wanted the distances, they would want them to do a histogram or their own percentiles, and would just compute the mean+-std themselves, and so wouldn't need the mean and std returned. 

But you're right, it is more intuitive to just return an additional argument. This is also consistent with the other `plan` functions then so that makes more sense.

Changed in the latest push.